api-extractor/src/definitions/ApiItem.ts

```diff
@@ -351,20 +357,20 @@ abstract class ApiItem {
    * an \@inheritdoc referencing ApiItemTwo, and ApiItemTwo has an \@inheritdoc refercing ApiItemOne then
    * we have a circular dependency and an error will be reported.
    */
-  public tryResolveReferences(): boolean {
+  public tryCompletingReferences(): boolean {
```

Comment:
Then your CompleteState could be `InitializationState.Completed`, `InitializationState.Incomplete`.